assets/src/edit-story/app/story/useStoryReducer/reducers/combineElements.js

```diff
@@ -50,9 +50,13 @@ import { removeAnimationsWithElementIds } from './utils';
  * @param {Object} payload Action payload
  * @param {string} payload.firstElement Element with properties to merge
  * @param {string} payload.secondId Element to add properties to
+ * @param {boolean} payload.isCopyAndPasteAction Is called from copy and paste
```

Comment:
This is pretty ugly. This argument should say something about intended change in functionality, not the source of the change. We might want the same functionality happen in other places, and if so, this naming will come back and haunt us.

Perhaps something like `shouldPurgeAnimations` – or `shouldRetainAnimations` and flip the logic (I prefer positively named booleans)?